portfolio/src/main/webapp/script.js

```diff
@@ -13,16 +13,76 @@
 // limitations under the License.
 
 /**
- * Adds a random greeting to the page.
+ * Adds a random piece of Uncle Iroh wisdom to the page
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function addIrohWisdom() {
+  const irohQuotes = 
+    ["Sharing tea with a fascinating stranger is one of life's true \
+      delights.",
+     "Hope is something you give yourself. That is the meaning of \
+      inner strength",
+     "Destiny is a funny thing. You never know how things are going to work \
+      out.", 
+     "It is usually best to admit mistakes when they occur, and to seek to \
+      restore honor.",
+     "While it is always best to believe in onself, a little help from \
+      others can be a great blessing.",
+     "Pride is not the opposite of shame, but its source. True humility \
+      is the only antidote to shame.",
+     "Life happens wherever you are, whether you make it or not."];
 
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+  // Pick random piece of Iroh wisdom
+  const irohQuote = irohQuotes[Math.floor(Math.random() * irohQuotes.length)]
 
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+  // Add it to the page
+  const irohContainer = document.getElementById('iroh-container');
+  irohContainer.innerText = irohQuote;
 }
+
+/**
+ * Generates a URL for a random image in the images directory and adds an img
+ * element with that URL to the page.
+ */
+function randomizeImage() {
+  // The images directory contains 13 images, so generate a random index between
+  // 1 and 13.
+  const imageIndex = Math.floor(Math.random() * 4) + 1;
```

Comment:
This will result in values between 1 and 4. The comment above mentions that it should be between 1 and 13. Please update.